syntax = "proto3";

package suggest;

import "google/api/annotations.proto";

service Suggest{
  rpc Answer(SuggestRequest) returns (SuggestResponse) {
    option (google.api.http) = {
      post: "/v1/suggest"
      body: "*"
    };
  }
}

message SuggestRequest {
  string input = 1;
}

message Suggest_answer{
 string text = 1;
 uint32 position = 2;
}

message SuggestResponse{
  repeated Suggest_answer sugg = 1;
}